test/k8sT/Policies.go

```diff
@@ -1316,6 +1319,312 @@ var _ = SkipDescribeIf(func() bool {
 
 	})
 
+	Context("Basic Policy Metrics Test", func() {
+		var (
```

Comment:
Another thing to consider is whether we want to use our ginkgo based tests for metric testing - this doesn't really integrate anything from k8s side. Maybe metrics should be tested via `cilium-cli`?

Reply:
Thank you so much @nebril , Yes agree with you.
Shall I raise another issue for metric testing in cilium-cli and track from there kindly suggest.